### DISCUSSION TITLE
How to change font size in i3wm config?

### DISCUSSION THREAD

**Discussion by jandono5** [i3/i3#6116] [Q&A]
I've recently migrated from linuxmint to archlinux using typecraft's excellent video series but I'm having issues using i3-wm. I simply want to increase the font size in i3's config but I'm not able to update the current setting. Even when the font size is increased, restarting the config using mod+shift+r does nothing. I've copied /etc/i3/config into ~/.config/i3/config but I'm still unable to change the config file.

If I do make changes to the config file then try to go back into it I get thrown 'E325: ATTENTION' followed by 'Found a swap file by the name'...

The i3 config is the default, an example found at https://github.com/i3/i3/blob/next/etc/config 

Any help is much appreciated! Again, bit of a newbie so apologies if this is a very stupid question.

**Comment by Leenuus** [comment]
> 'E325: ATTENTION' followed by 'Found a swap file by the name'

This seems a vim/neovim error, not i3's fault. You can simply delete the swap file vim/neovim said, then you can edit the config file, or it keeps prompting you every time you try to open and edit the same file with vim/neovim. Read the error messages, they must indicate the path of the swap file.

If you don't know how to delete the swap file, you can open your i3 config with other editor like a GUI one.
